src/js/components/award/details/DetailsTabBar.jsx

```diff
@@ -10,46 +10,20 @@ import DetailsTabItem from './DetailsTabItem';
 const propTypes = {
     activeTab: React.PropTypes.string,
     clickTab: React.PropTypes.func,
-    type: React.PropTypes.string
+    tabOptions: React.PropTypes.array
 };
 
-const tabOptions = [
-    {
-        label: 'Transaction History',
-        code: 'transaction',
-        disabled: false
-    },
-    {
-        label: 'Sub-Awards',
-        code: 'subaward',
-        disabled: false
-    },
-    {
-        label: 'Financial System Details',
-        code: 'financial',
-        disabled: false
-    }
-];
-
 export default class DetailsTabBar extends React.Component {
     render() {
-        const tabsSet = tabOptions.slice();
-        if (this.props.type === 'contract') {
-            tabsSet.push({
-                label: 'Additional Details',
-                code: 'additional',
-                disabled: false
-            });
-        }
-        const tabs = tabsSet.map((tab) => (
+        const tabs = this.props.tabOptions.map((tab) => (
             <DetailsTabItem
                 {...tab}
                 active={tab.code === this.props.activeTab}
```

Comment:
The tabs are no longer highlighting because this needs to be changed over to `internal`